modules/appeals_api/app/serializers/appeals_api/higher_level_reviews/v0/higher_level_review_serializer_with_pii.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module AppealsApi::HigherLevelReviews::V0
+  class HigherLevelReviewSerializerWithPii
+    include JSONAPI::Serializer
+    set_key_transform :camel_lower
+    set_type :higherLevelReview
+    attributes :status
+    attribute :code, if: proc { |hlr| hlr.status == 'error' }
+    attribute :detail, if: proc { |hlr| hlr.status == 'error' }
+    # These names are required by Lighthouse standards
+    attribute :createDate, &:created_at
+    attribute :updateDate, &:updated_at
+    # Only return form_data for created records
+    attribute :form_data, if: proc { |record| record.saved_change_to_id? }
+  end
+end
```

Comment:
This is the new serializer, which includes PII in form_data, but is otherwise the exact same as the one without PII. It's used at the `create` endpoint.

I tried to use inheritance to make this very simple, but for some reason, that did not work. I had to actually copy and paste the exact same content from the other serializer. :(